### DISCUSSION TITLE
Working with soft-deletes

### DISCUSSION THREAD

**Discussion by underdpt** [filamentphp/filament#2727] [Ideas]
Hello,

I think adding built-in support for soft deleted records would be very cool. I'm thinking maybe:

- Predefined table filter to show soft deleted items (show active records only / deleted records only / all records)
- Actions to undelete soft deleted items (and maybe trashing)
- Ability to edit a soft-deleted item, instead of showing a 404 error, show the record but only allow to undelete it (all inputs disabled)

What do you think?

**Comment by danharrin** [comment]
Soft-deleting features have now been released, you can pass `--soft-deletes` to the `make:filament-resource` and `make:filament-relation-manager` commands
